recipes/gpi_core/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set version = "2.0.0rc" %}
+{% set build_num = "0" %}
+{% set sha256 = "8dc97c93e48f57febc67773107276dbbdadb8c8effa75bea18a475c07266442a" %}
+
+package:
+  name: gpi_core
+  version: {{ version }}
+
+source:
+  url: https://github.com/gpilab/core-nodes/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  folder: repo_contents
+
+build:
+  number: {{ build_num }}
+  detect_binary_files_with_prefix: True
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}  # [linux]
+    - {{ compiler('cxx') }}  # [linux]
+    - gcc  # [mac]
```

Comment:
Do you need `gcc` on macOS? As far as I know it's deprecated and unmaintained in favour.

Reply:
Probably not - I think I misidentified this earlier as causing the osx build to work. I'll take it out when pushing up changes for your other suggestions, thanks for catching this.

Reply:
Looks good in the latest build.